public/content/en/vibed/deploy-on-heroku.md

```diff
@@ -0,0 +1,186 @@
+# Deploy on Heroku
+
+### Pre-requirements
+
+- Heroku [account]((https://signup.heroku.com/login))
+- [Git](https://git-scm.com/) installed
+- Your application should compile without error using dmd (`dub build`). 
+
+### 1 : Setup the app
+
+Heroku needs to know how to communicate with a deployed application. 
+Hence a global `PORT` environment variable is provided which needs to be injected into an application and 
+it should bind and listen to this port. 
+For development a default port (here __8080__) should be set:
+
+```d
```

Comment:
Does this work actually when you compile the dlang-tour?`I am just wondering because other code examples have an indent of 4 spaces for code examples and didn't know that the markdown parser of vibe.d supports this syntax.